src/bin/server.ts

```diff
@@ -12,6 +14,7 @@ interface Args {
 const expectedConfig = ['_', 'config', 'mock-driver'];
 
 (async () => {
+  // @ts-ignore
```

Comment:
We add types and then have to ignore the only place where the type are used? :D

Reply:
This is temporary until I found how to fix the error typescript is showing